umh.docs.umh.app/content/en/docs/production-guide/security/firewall-rules.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Firewall Rules"
+menuTitle: "Firewall Rules"
+description: |
+  This page describes how to setup firewall rules for the UMH instances.
+content_type: task
+weight: 50
+---
+
+<!-- overview -->
+
+To guarantee the security and seamless operation of your UMH instances, proper firewall rules must be configured.
```

Comment:
I would phrase it more on the line of "this page describes what to do if your enterprise network is closed and requires specific permissions"